src/main/webapp/app/pages/curation/CurationPage.tsx

```diff
@@ -171,7 +233,18 @@ const CurationPage = (props: ICurationPageProps) => {
                           }))}
                         />
                         <RealtimeCheckedInputGroup
-                          groupHeader="Mutation Effect"
+                          groupHeader={
+                            <>
+                              <span style={{ marginRight: '8px' }}>Mutation Effect</span>
```

Comment:
We can use bootstrap margin class in the places where we use `8px`. Or is there some alignment requirements where it must be 8px?

Reply:
Nope, switched to mr-2